coresimd/src/x86/i586/sse2.rs

```diff
@@ -744,51 +750,51 @@ pub unsafe fn _mm_set_epi16(
 pub unsafe fn _mm_set_epi8(
     e15: i8, e14: i8, e13: i8, e12: i8, e11: i8, e10: i8, e9: i8, e8: i8,
     e7: i8, e6: i8, e5: i8, e4: i8, e3: i8, e2: i8, e1: i8, e0: i8,
-) -> i8x16 {
+) -> __m128i {
     #[cfg_attr(rustfmt, rustfmt_skip)]
-    i8x16::new(
+    mem::transmute(i8x16::new(
```

Comment:
I guess we still need the portable vector types to implement some of the intrinsics, at least for the constructors.